packages/@uppy/companion/test/__tests__/companion.js

```diff
@@ -164,7 +184,7 @@ describe('download provdier file', () => {
       .set('uppy-auth-token', token)
       .set('Content-Type', 'application/json')
       .send({
-        endpoint: 'http://master.tus.com/files',
+        endpoint: 'http://master.tus.io/files',
```

Comment:
Wont it be confusing to see a tus endpoint while this is about s3 multipart? Even if it doesn’t matter for the test, folks will see this and draw bad conclusions or walk away in a confused state

Reply:
Yeah, I basically seconded that and meant to say “let’s use a random url in that case”, should have phrased it better.

Reply:
fixed on [master](https://github.com/transloadit/uppy/commit/73f2aaea37fc5cb5e03e0e1cef23a06785dd4382)